templates/web/panels/panel.tt.html

```diff
@@ -106,8 +106,30 @@ <h5>[% panel.title_element.subtitle %]</h5>
                         <div class="large-8 small-12 columns">
                 [% END %]
 
+                [% IF panel_group_element.icon_url.defined %]
+                <img src="[% panel_group_element.icon_url %]"
+                    style="[% IF panel_group_element.icon_size == 'very_small' %]height: 24px;[% ELSIF panel_group_element.icon_size == 'small' %]height:36px;[% ELSE %]height:72px;[% END %]float:left;margin-right:1rem;"
```

Comment:
couldn't we factor this kind of logic in a function ?

like `choose_size(panel_group_element.icon_size, "24px", "36px", "72px")`

Reply:
It's an option. Or in that case it might be simpler to just create CSS classes, and make the size part of the class name.

Reply:
Filed https://github.com/openfoodfacts/openfoodfacts-server/issues/9752 so that we can take care of it later.